packages/react/src/lib/mergeThemes.ts

```diff
@@ -53,8 +54,17 @@ export const mergeComponentStyles = (
       const originalTarget = partStylesPrepared[partName]
       const originalSource = partStyle
 
-      partStylesPrepared[partName] = styleParam => {
-        return _.merge(callable(originalTarget)(styleParam), callable(originalSource)(styleParam))
```

Comment:
this is causing some merging of styles with null, creating unnecessary instances. the perf improvement on the custom toolbar was around 5%